packages/mitex/specs/latex/standard.typ

```diff
@@ -53,6 +53,8 @@
   section: define-cmd(1, alias: "#heading(level: 1)"),
   subsection: define-cmd(1, alias: "#heading(level: 2)"),
   subsubsection: define-cmd(1, alias: "#heading(level: 3)"),
+  footnote: define-cmd(1, alias: "#footnote"),
+  cite: define-cmd(1, alias: "#mitexcite", handle: it => cite(label(get-tex-str(it)))),
```

Comment:
TBH, i don't really understand what is alias and handle, i just randomly copy code from this file. please check

Reply:
lgtm

Reply:
See https://github.com/mitex-rs/mitex/blob/002573384b7b184106c5a0cd5039e6ddbc812e1a/crates/mitex-spec/src/lib.rs#L152-L154

The alias is the name of the typst handle as well as the code converted from a latex command. The handle is put into the mitex scope and get invoked by name `alias`.